.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+    pull_request:
+      branches: 
+        - '**'
+
+jobs:
+  build:
+
+    env:
+      APP_ENV: test
+    
+    strategy:
+      matrix:
+        php: [ '7.3', '7.4' ]
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    
+    - name: Set up PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{matrix.php}}
+        coverage: none
+
+    - name: Install wikimedia-ocr
+      run: |
+        composer install
+        npm run build
+
+    - name: Test
+      run: |
+        composer test
```

Comment:
There's no `test` command defined in composer.json yet, but we should probably add one. So in this same PR you could add this to composer.json:
```
{
    ...
    "scripts": {
        ...
        "test": [
            "./vendor/bin/phpcs -s -p ."
        ]
    }
}
```

Reply:
done. Thank you!